ddln_common.proto

```diff
@@ -21,3 +21,9 @@ message Pose {
     optional Point3d position = 1;
     optional Quaternion orientation = 2;
 }
+
+message LatLongAlt {
+    optional double latitude_rad = 1;
```

Comment:
nit: it is useful to keep this quantities readable in communication - I often run dummy server or client and just print the protobuf for inspection. For those reasons I'd prefer to have degrees here.

Reply:
Consider it done